summingbird-online/src/test/scala/com/twitter/summingbird/online/PlannerSpec.scala

```diff
@@ -62,6 +63,75 @@ class PlannerSpec extends WordSpec {
   def arbSource1 = sample[Producer[Memory, Int]]
   def arbSource2 = sample[KeyedProducer[Memory, Int, Int]]
 
+  /*
+ * Test Case : A simple src.map.summer topology with FMMergeableWithSource opt-in functionality[ No FlatMappedProducer ].
```

Comment:
This comment could use some cleanup.  Not sure why [ is there, but we probably want to highlight in the comment (if not also in the test) that we'd usually get a `FlatMapNode` as the result of a `IdentityKeyedProducer`.